### PR TITLE
feat: Add Stack Pack for Gatsby

### PR DESCRIPTION
**Pull request by LekoArts**
Hi, Gatsby maintainer here 👋 

This PR adds a Stack Pack for Gatsby, mainly recommending our [Image Plugin](https://www.gatsbyjs.com/docs/how-to/images-and-media/using-gatsby-plugin-image/) and [Script API](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-script/).

I reused some wording from the Next.js PR: https://github.com/GoogleChrome/lighthouse-stack-packs/pull/66

### CODE REVIEW

**Review by connorjclark — CHANGES_REQUESTED** [review]
There's a failing test to address FYI

File: packs/gatsby.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const icon = `data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 28 28"><circle cx="14" cy="14" r="14" fill="#639"/><path fill="#fff" d="M6.2 21.8C4.1 19.7 3 16.9 3 14.2L13.9 25c-2.8-.1-5.6-1.1-7.7-3.2zm10.2 2.9L3.3 11.6C4.4 6.7 8.8 3 14 3c3.7 0 6.9 1.8 8.9 4.5l-1.5 1.3C19.7 6.5 17 5 14 5c-3.9 0-7.2 2.5-8.5 6L17 22.5c2.9-1 5.1-3.5 5.8-6.5H18v-2h7c0 5.2-3.7 9.6-8.6 10.7z"/></svg>`;
+
+const UIStrings = {
+  /** Additional description of a Lighthouse audit that tells the user how they can remove unused CSS rules by configuring the Gatsby plugin `gatsby-plugin-purgecss` which sets up PurgeCSS */
+  'unused-css-rules': 'Consider setting up `PurgeCSS` in `Gatsby` configuration to remove unused rules from stylesheets. [Learn more](https://purgecss.com/plugins/gatsby.html).',
```
[the remark this answers is not in the document]
Comment (LekoArts) [reply packs/gatsby.js]: This was copied from the Next.js piece - but I changed it to "use" now 👍

File: packs/gatsby.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+const icon = `data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 28 28"><circle cx="14" cy="14" r="14" fill="#639"/><path fill="#fff" d="M6.2 21.8C4.1 19.7 3 16.9 3 14.2L13.9 25c-2.8-.1-5.6-1.1-7.7-3.2zm10.2 2.9L3.3 11.6C4.4 6.7 8.8 3 14 3c3.7 0 6.9 1.8 8.9 4.5l-1.5 1.3C19.7 6.5 17 5 14 5c-3.9 0-7.2 2.5-8.5 6L17 22.5c2.9-1 5.1-3.5 5.8-6.5H18v-2h7c0 5.2-3.7 9.6-8.6 10.7z"/></svg>`;
```
Comment (connorjclark) [inline packs/gatsby.js:1]:
```suggestion
const icon = `data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 28 28"><circle cx="14" cy="14" r="14" fill="%23639"/><path fill="%23fff" d="M6.2 21.8C4.1 19.7 3 16.9 3 14.2L13.9 25c-2.8-.1-5.6-1.1-7.7-3.2zm10.2 2.9L3.3 11.6C4.4 6.7 8.8 3 14 3c3.7 0 6.9 1.8 8.9 4.5l-1.5 1.3C19.7 6.5 17 5 14 5c-3.9 0-7.2 2.5-8.5 6L17 22.5c2.9-1 5.1-3.5 5.8-6.5H18v-2h7c0 5.2-3.7 9.6-8.6 10.7z"/></svg>`;
```

**Review by connorjclark — APPROVED** [review]

**Comment by connorjclark** [comment]
Thanks! Expect this in the next Lighthouse release.

**Comment by LekoArts** [comment]
Awesome, thank you!

**Pull request merged**